### DISCUSSION TITLE
Sprite scale while pushing to screen

### DISCUSSION THREAD

**Discussion by FrankHovis** [Bodmer/TFT_eSPI#1005] [New processor/display support or capability request] [answered]
Is there any way a sprite can be scaled [up] while pushing to screen? Something along the lines of

spr.pushSprite(x, y, xscale, yscale) ; // where xscale=2 draws sprite twice X size.

**Comment by Bodmer** [comment, marked as answer]
No  though that is an interesting idea for a new feature.
